x-pack/plugins/cases/public/components/create/description.test.tsx

```diff
@@ -55,7 +52,7 @@ describe.skip('Description', () => {
       </MockHookWrapperComponent>
```

Comment:
I think the `FormTestComponent` from `x-pack/plugins/cases/public/common/test_utils.tsx` can(should?) be used instead of `MockHookWrapperComponent`. I haven't used it before but I think I saw it in one of @cnasikas flaky test fixes.

Reply:
It seems `MockHookWrapperComponent ` is a way to mock `form` when you want to test `schema`. Currently `FormTestComponent` doesn't take any schema. 

However it is a good idea to update `FormTestComponent` util to accept form schema as well. I can take a look and update it in a separate PR.

Reply:
Ah alright 👍 at first glance I thought here you just needed the default values.